packages/schema/tests/idl.rs

```diff
@@ -109,10 +109,7 @@ fn test_query_responses() {
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema, QueryResponses)]
 #[serde(rename_all = "snake_case")]
-pub enum QueryMsgWithGenerics<T: std::fmt::Debug>
-where
-    T: JsonSchema,
-{
+pub enum QueryMsgWithGenerics<T: std::fmt::Debug> {
```

Comment:
Do we still need to keep `: std::fmt::Debug`? Isn't that trait bound added via the implementation the same way as `JsonSchema`?

Reply:
The implementation only adds `JsonSchema` right now. It doesn't need a `Debug` trait bound. If you remove this `Debug` trait bound, the test still passes.

I added the `Debug` trait bound to make sure the macro not only works with type parameters, but also with trait bound type parameters.

Reply:
Thank you! Then just the CHANGELOG and 🐎

Reply:
It's not that important and does make the test confusing though. I could split it into multiple tests?

Reply:
> It's not that important and does make the test confusing though. I could split it into multiple tests?

I'm fine either way. My original question was ansered and it makes sense.